### DISCUSSION TITLE
declarative-user-profile: attributes dont show up on user profile

### DISCUSSION THREAD

**Discussion by ryan-morris** [keycloak/keycloak#15575] [Q&A] [answered]
I've enabled `declarative-user-profile` and any attributes I add under the `User Profile` area of `Realm Settings` show up on the registration form but not on the `Personal Info` page where users can manage their information. 

Is there something I am overlooking or is that part of the feature not yet implemented?

**Comment by sschu** [comment, marked as answer]
It is not yet implemented but should be on the list.
